connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala

```diff
@@ -240,6 +241,12 @@ object CassandraConnectorConf extends Logging {
     deprecatedSince = "DSE 6.0.0"
   )
 
+  val ResolveContactPoints = ConfigParameter[Boolean](
+    name = "spark.cassandra.connection.resolveContactPoints",
+    section = ReferenceSection,
+    default = true,
+    description = """Should we resolve contact points to IPs on start (true, false)""")
```

Comment:
We can be more helpful here, Perhaps "Resolves contact points on every connection. Helpful for usage with Kubernetes or other systems with Dynamic Endpoints which may change while the application is running.

Reply:
done